Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -345,6 +344,23 @@ GDCMImageIO::Read(void * pointer)
     len = len * outputpt.GetPixelSize() / pixeltype.GetPixelSize();
   }
 
+  // Y'CbCr to RGB
+  // GDCM 3.0.3
+  const bool ybr =
+    m_NumberOfComponents == 3 &&
+    (pi == gdcm::PhotometricInterpretation::YBR_FULL ||
+     pi == gdcm::PhotometricInterpretation::YBR_FULL_422) &&
+    (pixeltype == gdcm::PixelFormat::UINT8 ||
+     pixeltype == gdcm::PixelFormat::INT8);
+  if (ybr && m_ReadYBRtoRGB)
```

Comment:
As mentionned before PR #1310 does make use of the gdcm filter for this purpose. Please keep the implementation in sync with GDCM design.

Reply:
What is here "not in sync with GDCM design"?
```
static void YCbCr_to_RGB(unsigned char * p, const size_t len)
{
  for (size_t x = 0; x < len; x += 3)
  {
    const unsigned char in[] = { p[x], p[x + 1], p[x + 2] };
    gdcm::ImageChangePhotometricInterpretation::YBR2RGB<unsigned char>(&p[x], in);
  }
}
```